pkg/sources/github/github.go

```diff
@@ -150,6 +155,76 @@ func (s *Source) Init(aCtx context.Context, name string, jobID, sourceID int64,
 	return nil
 }
 
+func (s *Source) isPublic(repoURL string) (visability source_metadatapb.Visibility) {
+	visability, exists := s.publicMap[repoURL]
```

Comment:
**Typo nit:** `visability` -> `visibility`

Reply:
Also, I'm not sure if this variable shadows the named return variable or not..

To be safe, maybe we can inline the if:

```go
if visibility, ok := s.publicMap[repoURL]; ok {
    return visibility
}
```